src/components/evm/UserInfo.vue

```diff
@@ -0,0 +1,203 @@
+<script lang="ts">
+import { defineComponent } from 'vue';
+import { useAccountStore, getAntelope } from 'src/antelope';
+import InlineSvg from 'vue-inline-svg';
+
+const accountStore = useAccountStore();
+const ant = getAntelope();
+
+export default defineComponent({
+    name: 'UserInfo',
+    components: {
+        InlineSvg,
+    },
+    methods: {
+        copyAddress() {
+            if (!accountStore.loggedAccount.isNative && accountStore.isAuthenticated) {
+                const address = accountStore.loggedEvmAccount.account;
+                navigator.clipboard.writeText(address.toUpperCase());
```

Comment:
this should actually be checksum, not uppercase

Reply:
What do you mean by that? can you suggest the correct code?